src/utils.rs

```diff
@@ -3,7 +3,7 @@
 /* convert a path to a NUL-terminated Vec<u8> suitable for use with C functions */
 #[cfg(all(
     not(feature = "unknown-ci"),
-    any(target_os = "linux", target_os = "android", target_vendor = "apple")
+    any(target_os = "linux", target_os = "android")
```

Comment:
As a follow-up once this PR is merged, we can move this function into the `linux` folder directly since it's only used by linux targets.

Reply:
Done in https://github.com/GuillaumeGomez/sysinfo/pull/873.

Reply:
Thanks for handling that!

Reply:
:+1: